src/Components/EditProduct/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Form from './Form';
+
+const DisplayForm = (props) => {
+	return(
+		<div>
+			<Form />

```

Comment:
I don't think you really need 4 files for this. You can use just a single file, ProductEdit.js. I would even place it under Home folder, but not sure, asking @brurez for help here.